package.json

```diff
@@ -78,6 +81,10 @@
         "webpack-dev-server": "^3.11.2",
         "worker-loader": "^2.0.0"
     },
+    "peerDependencies": {
+        "react": "^16.9.0",
+        "react-dom": "^16.9.0"
+    },
```

Comment:
following recommended best practices for including react as peerDependency and devDependency, not a "dependency"